src/components/RegisterForm/RegisterForm.jsx

```diff
@@ -77,100 +93,128 @@ const RegisterForm = () => {
   });
 
   return (
-    <form onSubmit={formik.handleSubmit} autoComplete="off">
-      <ItemWrapp>
-        <StyledLabel
-          color={{
-            error: formik.errors.userName,
-            touched: formik.touched.userName,
-          }}
-          htmlFor="Name"
-        >
-          Name
-        </StyledLabel>
-        <StyledInput
-          id="userName"
-          name="userName"
-          type="text"
-          onChange={formik.handleChange}
-          value={formik.values.userName}
-          placeholder="Enter your name"
-          color={{
-            error: formik.errors.userName,
-            touched: formik.touched.userName,
-          }}
-        />
+    <Background>
+      <RegisterFormContainer>
+        <SignInFormStyles onSubmit={formik.handleSubmit} autoComplete="off">
+          <SignInTitle>Create account</SignInTitle>
+          <SignInText>
+            Already have an account? <LoginLink>Log in</LoginLink>
+          </SignInText>
+          <ItemWrapp>
+            <StyledLabel
+              color={{
+                error: formik.errors.userEmail,
+                touched: formik.touched.userEmail,
+              }}
+              htmlFor="email"
+            />
+            <StyledInput
+              id="userEmail"
+              name="userEmail"
+              type="email"
+              onChange={formik.handleChange}
+              value={formik.values.userEmail}
+              placeholder="E-mail"
+              color={{
```

Comment:
You can check if input has any errors and has been touched in this component and pass appropriate prop to the styled component so that you could avoid havy logic inside.